DisguiseUnityRenderStream/Runtime/DisguiseRenderStream.cs

```diff
@@ -511,8 +520,27 @@ static Camera GetChannelCamera(string channel)
         }
 
         public static DisguiseRenderStream Instance { get; private set; }
+        
+        public static Action SceneLoaded { get; set; } = delegate { };
+        
+        public static Action StreamsChanged { get; set; } = delegate { };
 
         public StreamDescription[] Streams { get; private set; } = { };
+        
+        public IEnumerable<RenderTexture> InputTextures
+        {
+            get
+            {
+                if (LatestFrameData.scene > m_SceneFields.Length)
+                    return null;
```

Comment:
I think it's less burdensome on the client if you return `Enumerable.Empty<RenderTexture>()` here instead of `null`, which will require special handling.